platforms/quorum/configuration/roles/setup/geth/tasks/main.yaml

```diff
@@ -10,26 +10,19 @@
     path: "{{ bin_install_dir }}/geth"
   register: geth_stat_result
 
-  # This task checks if quorum repo directory exists
-- name: Check quorum repo dir exists
-  stat:
-    path: "{{ bin_install_dir }}/quorum"
-  register: quorum_stat_result
-
-# This task clones the quorum git repository
-- name: Clone the git repo
-  git:
-    repo: "{{ geth.repo }}"
-    version: master
-    force: yes
-    dest: "{{ bin_install_dir }}/quorum"
-  when: not ( quorum_stat_result.stat.exists and geth_stat_result.stat.exists )
+# This task creates the geth directory
+- name: Create geth directory
+  file:
+    path: "{{ bin_install_dir }}/geth"
+    state: directory
+  when: geth_stat_result.stat.exists == False
 
-# This task builds the geth binary
-- name: Make geth
-  make:
-    chdir: "{{ bin_install_dir }}/quorum"
-    target: geth
+# This task downloads the geth tar
+- name: Download geth tar 
+  get_url:
+    url: "https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.10.0-56dec25a.tar.gz"
+    dest: "{{ bin_install_dir }}/geth"
+    mode: 0777
```

Comment:
Do we need 777 mode? (Try to work with minimal permissions as all environments are not so flexible in giving full access)